From 2748f99730e699b2597653ddef1c4109cae976b2 Mon Sep 17 00:00:00 2001
From: Brandon Philips <brandon@ifup.co>
Date: Tue, 1 Apr 2014 16:35:28 -0700
Subject: [PATCH] fix(google-startup-scripts): use
 ${GOOGLE_STARTUP_SCRIPTS_PREFIX} in onboot

We can't install these scripts into /usr/share/google. So, take a PREFIX
environment variable.
---
 google-startup-scripts/usr/share/google/onboot | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/google-startup-scripts/usr/share/google/onboot b/google-startup-scripts/usr/share/google/onboot
index d740d07..f65b5ab 100755
--- a/google-startup-scripts/usr/share/google/onboot
+++ b/google-startup-scripts/usr/share/google/onboot
@@ -33,7 +33,7 @@ else
     declare -r LOGGER=/bin/logger
 fi
 
-declare -r BOTO_SETUP_SCRIPT=/usr/share/google/boto/boot_setup.py
+declare -r BOTO_SETUP_SCRIPT=${GOOGLE_STARTUP_SCRIPTS_PREFIX}/boto/boot_setup.py
 
 declare -r CURL_RETRY_LIMIT=10
 declare -r CURL_TIMEOUT=10
@@ -105,13 +105,13 @@ function download_url() {
 
 function get_metadata_value() {
   local readonly varname=$1
-  /usr/share/google/get_metadata_value ${varname}
+  ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/get_metadata_value ${varname}
   return $?
 }
 
 function get_metadata_attribute() {
   local readonly varname=$1
-  /usr/share/google/get_metadata_value "attributes/${varname}"
+  ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/get_metadata_value "attributes/${varname}"
   return $?
 }
 
@@ -129,14 +129,14 @@ function run_command_with_retry() {
 }
 
 function first_boot() {
-  if [[ -x /usr/share/google/first-boot ]]; then
-    /usr/share/google/first-boot
+  if [[ -x ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/first-boot ]]; then
+    ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/first-boot
   fi
 }
 
 function virtionet_irq_affinity() {
-  if [[ -x /usr/share/google/virtionet-irq-affinity ]]; then
-    /usr/share/google/virtionet-irq-affinity
+  if [[ -x ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/virtionet-irq-affinity ]]; then
+    ${GOOGLE_STARTUP_SCRIPTS_PREFIX}/virtionet-irq-affinity
   fi
 }
 
-- 
1.8.5.2 (Apple Git-48)

